servicedesk.go

```diff
@@ -112,3 +117,107 @@ func (s *ServiceDeskService) RemoveOrganizationWithContext(ctx context.Context,
 func (s *ServiceDeskService) RemoveOrganization(serviceDeskID int, organizationID int) (*Response, error) {
 	return s.RemoveOrganizationWithContext(context.Background(), serviceDeskID, organizationID)
 }
+
+// AddCustomersWithContext adds customers to the given service desk.
+//
+// https://developer.atlassian.com/cloud/jira/service-desk/rest/api-group-servicedesk/#api-rest-servicedeskapi-servicedesk-servicedeskid-customer-post
+func (s *ServiceDeskService) AddCustomersWithContext(ctx context.Context, serviceDeskID int, acountIDs ...string) (*Response, error) {
```

Comment:
I created a PR to change `serviceDeskID int` to `serviceDeskID interface{}` because it should have actually been a string, but for the sake of not breaking other peoples use I opt'd for an interface. I would suggest also accepting string values.

See the PR here https://github.com/andygrunwald/go-jira/pull/419

Reply:
I would agree with @james-cast, Please update as requested